e2e/accounts/accounts_rpc_test.go

```diff
@@ -40,6 +40,9 @@ func (s *AccountsTestSuite) TestRPCEthAccounts() {
 }
 
 func (s *AccountsTestSuite) TestRPCEthAccountsWithUpstream() {
+	// FIXME(tiabc): Stop skipping after https://github.com/status-im/status-go/issues/424
+	s.T().Skip()
```

Comment:
There's no point of upstream e2e tests when all tests are run against StatusChain